pages/index.tsx

```diff
@@ -1,10 +1,31 @@
 import type { NextPage } from 'next';
 import styles from 'styles/Home.module.scss';
+import ComeWorkForYou from 'components/ComeWorkForYou';
+
+const PlaceholderImage = {
+    description: 'This is a placeholder',
+    url: 'https://source.unsplash.com/random/1600×900/?nature',
+    width: 1600,
+    height: 900,
+};
 
 const Home: NextPage = () => {
     return (
         <div className={styles.container}>
             <h1 className={styles.title}>Vercel Setup</h1>
+            <ComeWorkForYou image={PlaceholderImage}>
+                <strong>
+                    No parent company, no shareholders, just a team of equal
+                    owners.
+                </strong>
+                <br />
```

Comment:
**Suggestion:** I'd use margin-bottom instead of the line break elements here.

Reply:
This is placeholder text that is currently waiting on the rich text component integration, I will replace this in the future